src/dodal/devices/electron_analyser/util.py

```diff
@@ -0,0 +1,13 @@
+from dodal.devices.electron_analyser import EnergyMode
+
+
+def to_kinetic_energy(
+    value: float, value_mode: EnergyMode, excitation_energy: float
+) -> float:
+    return excitation_energy - value if value_mode == EnergyMode.BINDING else value
```

Comment:
nit: I'd flip these clauses around to be a little clearer on purpose

`return value if value_mode == EnergyMode.KINETIC else excitation - value`